### PR TITLE
Compile scanjob

### PR DESCRIPTION
**Pull request by gdetrez**
- Add a `projectdirectory` arg to compile page 
- Add a link to compilation in the ScanningJobPanel
- Add a system test that creates a scanjob, post a scan, click the new "compile" link and checks the project path and that the images is detected.

### CODE REVIEW

**Comment by skymandr** [comment]
That's a pretty heft coverage decrease... :-/

**Comment by gdetrez** [comment]
Whoops. Should be fixed.

File: tox.ini [hunk 1]
```diff
@@ -5,6 +5,7 @@ skipsdist = true
 [testenv]
 deps =
     -rrequirements.txt
+    blinker
```
Comment (local-minimum) [inline tox.ini]: What is this? Don't see it used.

Comment (gdetrez) [reply tox.ini]: It is used by flask signal extension mechanism which I used in the test to get the template context.

**Pull request merged**